internal/awsnuke/client.go

```diff
@@ -0,0 +1,116 @@
+package awsnuke
+
+import (
+	"context"
+
+	"github.com/gruntwork-io/gruntwork-cli/errors"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/credentials"
+	"github.com/aws/aws-sdk-go/aws/endpoints"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/ec2"
+)
+
+var DefaultRegions = []string{
```

Comment:
Does the AWS SDK not have a full list of regions?

Reply:
I will have to check. I only looked into it briefly and found these regions in the [endpoints pkg](https://docs.aws.amazon.com/sdk-for-go/api/aws/endpoints/#pkg-constants).